apps/web/src/app/(main)/(mdx)/privacy/page.mdx

```diff
@@ -18,96 +20,113 @@ Sensitive Information. We do not process sensitive information.
+
+* Business Transfers. We may share or transfer your information in connection with, or during negotiations of, any merger, sale of company assets, financing, or acquisition of all or a portion of our business to another company.
+* Affiliates. We may share your information with our affiliates, in which case we will require those affiliates to honor this privacy notice. Affiliates include our parent company and any subsidiaries, joint venture partners, or other companies that we control or that are under common control with us.
+* Business Partners. We may share your information with our business partners to offer you certain products, services, or promotions.
+
+
+## 4. DO WE USE COOKIES AND OTHER TRACKING TECHNOLOGIES?
 
 _In Short: We may use cookies and other tracking technologies to collect and store your information._
 
 We may use cookies and similar tracking technologies (like web beacons and pixels) to access or store information. Specific information about how we use such technologies and how you can refuse certain cookies is set out in our Cookie Notice.
 
-## 5\. HOW DO WE HANDLE YOUR SOCIAL LOGINS?
+
+## 5. HOW DO WE HANDLE YOUR SOCIAL LOGINS?
 
 _In Short: If you choose to register or log in to our Services using a social media account, we may have access to certain information about you._
 
 Our Services offer you the ability to register and log in using your third-party social media account details (like your Facebook or Twitter logins). Where you choose to do this, we will receive certain profile information about you from your social media provider. The profile information we receive may vary depending on the social media provider concerned, but will often include your name, email address, friends list, and profile picture, as well as other information you choose to make public on such a social media platform.
 
 We will use the information we receive only for the purposes that are described in this privacy notice or that are otherwise made clear to you on the relevant Services. Please note that we do not control, and are not responsible for, other uses of your personal information by your third-party social media provider. We recommend that you review their privacy notice to understand how they collect, use, and share your personal information, and how you can set your privacy preferences on their sites and apps.
 
-## 6\. IS YOUR INFORMATION TRANSFERRED INTERNATIONALLY?
+
+## 6. IS YOUR INFORMATION TRANSFERRED INTERNATIONALLY?
 
 _In Short: We may transfer, store, and process your information in countries other than your own._
 
 Our servers are located in. If you are accessing our Services from outside, please be aware that your information may be transferred to, stored, and processed by us in our facilities and by those third parties with whom we may share your personal information (see "WHEN AND WITH WHOM DO WE SHARE YOUR PERSONAL INFORMATION?" above), in and other countries.
```

Comment:
update this too `Our servers are located in India. If you are accessing our Services from outside, please be aware that your information may be transferred to, stored, and processed by us in our facilities and by those third parties with whom we may share your personal information (see "WHEN AND WITH WHOM DO WE SHARE YOUR PERSONAL INFORMATION?" above), in India and other countries.`